.github/workflows/legacy-path-guard.yaml

```
@@ -1,35 +1,35 @@
-          node-version: 22
-          cache: npm
-
-      - name: Install root dependencies
-        run: npm install --ignore-scripts
-
-      - name: Fetch base branch
-        run: git fetch origin "${{ github.base_ref }}" --depth=1
-
-      - name: Guard migrated legacy paths
-        run: npm run ts-migration:guard -- --base "origin/${{ github.base_ref }}" --head HEAD
+# SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)